imsim/sky_model.py

```diff
@@ -108,8 +112,130 @@ def __call__(self, x, y):
         value at the CCD center.
         """
         return (self.a*x + self.b*y + self.c)/self.sky_level_center
+    
+class CCD_Fringing:
+    """
+    Class generates normalized fringing map. 
+    
+    The function call operator returns the normlized fringing level
+    as a function of pixel coordinates relative to the value at the 
+    CCD center.
+    """
+    def __init__(self,img_wcs,boresight,seed,spatial_vary):
+        """
+        Parameters
+        ----------
+        img_wcs : 
+            WCS of the current image.
+        boresight : `array`
+            Array stores the center RA/DEC of boresight.
+        seed : `int`
+            Random seed number for each CCD. 
+            This number should be the det_num for the current sensor
```

Comment:
I don't think `det_num` should be used as the seed. It's reused across different cameras, so not unique to a particular CCD, and one can make yaml configs for a given camera where the det_num to CCD mapping isn't preserved.  If we switch to using the GalSim random number generator, then we can use the detector serial number as the seed, which would have the desired behavior.   Also the detector serial numbers will tell us which kind of CCD we are considering, i.e., e2V vs ITL from the first three letters of the serial number, so we can account for different fringing behavior for the different types that way.

Reply:
Using serial number sounds good. Do you know where can I access that info?

Reply:
Here's some example code:
```
import imsim
camera = imsim.get_camera()

serial_number = camera[det_name].getSerial()
```

Reply:
Thank you. I will adjust the code accordingly.

Reply:
The final line in the doc for seed is now incorrect.